File: src/foggyhollow/timetable/TimetableEntry.java
package foggyhollow.timetable;

public class TimetableEntry
{
    private String trainNumber;
    private String trainName;
    private String destination;
    private String arLv;
    private String scheduleTime;
    private String direction;
    
    public TimetableEntry()
    {
	
    }
    public String getTrainNumber()
    {
        return trainNumber;
    }

    public void setTrainNumber(String trainNumber)
    {
        this.trainNumber = trainNumber;
    }

    public String getTrainName()
    {
        return trainName;
    }

    public void setTrainName(String trainName)
    {
        this.trainName = trainName;
    }

    public String getDestination()
    {
        return destination;
    }

    public void setDestination(String destination)
    {
        this.destination = destination;
    }

    public String getArLv()
    {
        return arLv;
    }

    public void setArLv(String arLv)
    {
        this.arLv = arLv;
    }

    public String getScheduleTime()
    {
        return scheduleTime;
    }

    public void setScheduleTime(String scheduleTime)
    {
        this.scheduleTime = scheduleTime;
    }

    public String getDirection()
    {
        return direction;
    }

    public void setDirection(String direction)
    {
        this.direction = direction;
    }    
}

File: layouts/DevilsGulch/scripts/TimeTableEntry.py
class TimeTableEntry(object):

    def setDirection(self, direction):
        self.direction = direction;
    def getDirection(self):
        return self.direction

    def setTrainNumber(self, number):
        self.trainNumber = number
    def getTrainNumber(self):
        return self.trainNumber

    def setTrainName(self, name):
        self.trainName = name
    def getTrainName (self):
        return self.trainName

    def setScheduleTime(self, schedTime):
        self.scheduleTime = schedTime
    def getScheduleTime(self):
        return self.scheduleTime

    def setArLv(self, arlv):
        self.arlv = arlv
    def getArLv(self):
        return self.arlv

    def setDestination(self, destination):
        self.destination = destination
    def getDestination(self):
        return self.destination

    def setRemark(self, remark):
        self.remark = remark
    def getRemark(self):
        return self.remark

    def __init__(self):
        self.direction = "UNK"
        self.trainNumber = "-1"
        self.trainName = "UNK"
        self.scheduleTime = "UNK"
        self.arlv = "UNK"
        self.destination = "UNK"
        self.remark = ""
